devo/sender/data.py

```diff
@@ -188,12 +190,22 @@ def __connect_ssl(self):
                 if self._sender_config.key is not None \
                         and self._sender_config.chain is not None \
                         and self._sender_config.cert is not None:
-                    self.socket = ssl.wrap_socket(
-                        self.socket,
-                        keyfile=self._sender_config.key,
-                        certfile=self._sender_config.cert,
-                        ca_certs=self._sender_config.chain,
-                        cert_reqs=ssl.CERT_REQUIRED)
+
+                    context = ssl.create_default_context(
+                        cafile=self._sender_config.chain)
+
+                    if self._sender_config.sec_level is not None:
+                        context.set_ciphers(
+                            "DEFAULT@SECLEVEL={!s}"
+                            .format(self._sender_config.sec_level))
```

Comment:
Do we have a logger here? I think that is a good idea to log a WARNING message when the sec_level is different than the default

Reply:
Added on warning message